bloch_sphere/animate_bloch.py

```diff
@@ -142,6 +143,15 @@ def apply_gate_list(self, gates, final_wait=True):
                     print(f'Error: Custom gate arguments contain invalid '
                           f'floats {gate}.')
                 continue
+            if re.match('R[x,y,z][,;]', gate[:3]):
```

Comment:
Can you make it lowercase or case insensitive?

Reply:
Done